2-mandatory/4-stocks.js

```diff
@@ -34,7 +34,18 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    
+    let arrayOfAverage = []
+    for(let i of closingPricesForAllStocks){
+        let total = 0;
+        for(let j in i){   
+        total += i[j];  
+        }
+        let a = total/i.length;
```

Comment:
I'm not a huge advocate for your variable naming here, you have `i`, `j`, `a` and `b` - Imagine if this code was 500 lines long, we'd need to study it closely to know what was happening.

Other than that, good job! There are a couple other ways to do this but you succeeded in completed the exercise.

Reply:
I really need to get used to it.